doc/administrator/administrate.rst

```diff
@@ -188,7 +131,8 @@ be the zoom limit.
 
 Print WMTS in high quality
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
-To print the layers in high quality, you should add the following ``Metadata``:
+To print the layers in high quality, we can get the image with a GetMap on the original WMS server.
```

Comment:
"we can get with a GetMap on the original WMS server."
-> "you can define that the image shall be retrieved with a GetMap on the original WMS server."